### PR TITLE
Main Menu V1

### PR DESCRIPTION
**Pull request by garenpham**
Created My Transfer page and route. Added the new route to ClientRoutes.js. Updated Menu styles. Change styles on Send Tokens page, making its title to be visible. Organized import orders in the import section on ClientRoutes.js.

### CODE REVIEW

**Comment by garenpham** [comment]
@OlhaD thank you for your feedback! I have linked the issue to this pull request.

File: src/components/layout/Menu/Menu.js [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+  DrawerStyled,
+  LogoStyled,
+} from './MenuStyled';
 
 const Menu = ({ open, handleDrawerClose, handleDrawerOpen }) => {
   const theme = useTheme();
 
   return (
     <>
-      <TopMenu handleDrawerOpen={handleDrawerOpen} open={open} />
```
Comment (OlhaD) [inline src/components/layout/Menu/Menu.js:15]: Why did you move the TopMenu code back to Menu.js? Is it a merge issue or you had some idea about it? I think it is good to have a separate component to keep them as thin and clean as possible. Also, the top menu contains the logout icon which is missing.

File: src/components/layout/Menu/MenuStyled.js [hunk 2]
```diff
@@ -1,60 +1,93 @@
...
     duration: theme.transitions.duration.leavingScreen,
   }),
-  overflowX: "hidden",
+  overflowX: 'hidden',
   width: `calc(${theme.spacing(8)} + 1px)`,
-  [theme.breakpoints.up("sm")]: {
+  [theme.breakpoints.up('sm')]: {
     width: `calc(${theme.spacing(8)} + 1px)`,
   },
 });
```
Comment (OlhaD) [inline src/components/layout/Menu/MenuStyled.js:32]: Do you use Prettier as a code formatter? I think it should allow double quotes. Just not to change the same imports with every commit made by different people

File: src/components/layout/Menu/MenuStyled.js [hunk 3]
```diff
@@ -1,60 +1,93 @@
...
+const DrawerHeaderStyled = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
   padding: theme.spacing(0, 1),
   // necessary for content to be below app bar
   ...theme.mixins.toolbar,
 }));
 
+const AppBarStyled = styled(MuiAppBar, {
```
Comment (OlhaD) [inline src/components/layout/Menu/MenuStyled.js:42]: The same here, it was a part of TopMenuStyled.js

File: src/components/layout/Menu/Menu.js (hunk 1 above)
Comment (garenpham) [reply src/components/layout/Menu/Menu.js]: It was my fault. I did not sync my fork with the latest update. I have done it now and turns out the MenuStyled does not have to change anything. Thank you for pointing that out!

File: src/components/layout/Menu/MenuStyled.js (hunk 3 above)
Comment (garenpham) [reply src/components/layout/Menu/MenuStyled.js]: Yes, I have fixed it now.

File: src/components/layout/Menu/MenuStyled.js (hunk 2 above)
Comment (garenpham) [reply src/components/layout/Menu/MenuStyled.js]: I have changed my Prettier settings to be matched with Greendstand code format.

File: src/components/layout/Menu/Menu.js (hunk 1 above)
Comment (garenpham) [reply src/components/layout/Menu/Menu.js]: @OlhaD Thank you for your insightful review. I have fixed my code according to your feedback. Can you please check them again and initiate the merge?

**Review by OlhaD — APPROVED** [review]

File: src/components/layout/Menu/Menu.js (hunk 1 above)
Comment (OlhaD) [reply src/components/layout/Menu/Menu.js]: Thank you for the fixes
